### DISCUSSION TITLE
Manga/Webcomics that aren't on Anilist

### DISCUSSION THREAD

**Discussion by Vodes** [metafates/mangal#134] [Q&A] [answered]
Would it be possible to download those without mangal adding a comicinfo to them? Might be somewhat of an edgecase tbh.

One such example would be The Beginning After the End which is available on plenty of manga sites

**Comment by metafates** [comment, marked as answer]
Hi! If I understand you correctly - you want to disable metadata fetching?

You can do it by executing this command

```shell
mangal config set -k metadata.fetch_anilist -v false
```

Or manually editing the config file

```toml
[metadata]
fetch_anilist = false
```

...Or by changing an environment variable `MANGAL_METADATA_FETCH_ANILIST`

---

If you want to disable ComicInfo.xml generation use this

```shell
mangal config set -k metadata.comic_info_xml -v false
```

**Reply by Vodes** [reply]
Would be cool to have it in inline, so it wouldn't mess up the other manga I'm downloading. For example if I were to add a cronjob for it inside of my kaizoku container

**Reply by metafates** [reply]
> Would be cool to have it in inline, so it wouldn't mess up the other manga I'm downloading. For example if I were to add a cronjob for it inside of my kaizoku container

I think you might want this inline flag

`-f, --fetch-metadata          Populate manga metadata`

**Reply by Vodes** [reply]
I'm not entirely sure what that does. From what I can tell, inline always adds comicinfos and series.json. I would love a flag to *not* do that

**Reply by metafates** [reply]
> I'm not entirely sure what that does. From what I can tell, inline always adds comicinfos and series.json. I would love a flag to *not* do that 

I see. Well, I can add that, sure.

But just in case you want to temporarily _disable_ metadata fetching you can use this flag like that

`--fetch-metadata=false`

That way it will still create both ComicInfo and series.json files (if specified so in config), but it won't ask anilist to fill missing fields, e.g. summary, tags or anything else

**Reply by Vodes** [reply]
Ah alright that makes sense. I'll try if that's enough for me 🙂
